arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_1-how-das-work-content.mdx

```diff
@@ -0,0 +1,10 @@
+A Data Availability Server (DAS) allows storage and retrieval of transaction data batches for an AnyTrust chain. It's the software that the members of the DAC run in order to provide the Data Availability service.
+
+DA servers accept time-limited requests to store data batches from the sequencer of an AnyTrust chain, and return a signed certificate promising to store that data during the established time. They also respond to requests to retrieve the data batches.
+
+Additionally, mirror DA servers can be run to help handle requests to retrieve data batches (however, they won't store batches). They may also provide archived data beyond the limited time that DA servers are required to store the data.
```

Comment:
Maybe '(however, they won't handle requests to store batches)', since they do store the batches but don't support the Sequencer API.